app/src/main/java/net/gsantner/markor/ui/hleditor/TextActions.java

```diff
@@ -619,6 +622,19 @@ protected boolean runCommonTextAction(final @StringRes int action) {
                 runRenumberOrderedListIfRequired();
                 return true;
             }
+            case R.string.tmaid_common_insert_snippet: {
+                SearchOrCustomTextDialogCreator.showInsertSnippetDialog(_activity, (snip) -> {
+                    _hlEditor.replaceCurrentSelection(StringUtils.interpolateEscapedDateTime(snip));
+                    _lastSnip = snip;
```

Comment:
Storing the last snippet in memory in each TextFormat instance. Not stored in appsettings this way. Doesn't matter if data is lost on rotation etc

Reply:
OK this also better than storting to settings, because if you use multiple windows you can have different "last actions" this way